portal-web/test/functional/com/liferay/portalweb/macros/MultiFactorAuthentication.macro

```diff
@@ -228,6 +253,37 @@ definition {
 			value1 = "Your one-time password has been sent by email.");
 	}
 
+	macro verifyCustomOTP {
+		if (isSet(specificURL)) {
+			Navigator.openSpecificURL(url = "${specificURL}");
+		}
+		else {
+			Navigator.openURL();
+		}
+
+		SignIn._signIn(
+			userEmailAddress = "${userEmailAddress}",
+			userPassword = "${userPassword}");
+
+		AssertElementPresent(locator1 = "MultiFactorAuthentication#CUSTOM_MFA_TITLE");
+		AssertElementPresent(locator1 = "MultiFactorAuthentication#CUSTOM_MFA_DESCRIPTION");
+		AssertElementPresent(locator1 = "MultiFactorAuthentication#EMAIL_OTP_LINK");
```

Comment:
These lines should be spaced out even though order doesn't matter

Reply:
Done